lnbits/decorators.py

```diff
@@ -29,20 +30,20 @@ def __init__(
         self._key_type = "invoice"
         self._api_key = api_key
         if api_key:
-            self.model: APIKey = APIKey(
+            key = APIKey(
                 **{"in": APIKeyIn.query},
                 name="X-API-KEY",
                 description="Wallet API Key - QUERY",
             )
         else:
-            self.model: APIKey = APIKey(
+            key = APIKey(
                 **{"in": APIKeyIn.header},
                 name="X-API-KEY",
                 description="Wallet API Key - HEADER",
             )
-        self.wallet = None
```

Comment:
Why do you remove this? See below. Is the default value `None`? Does it matter?

Reply:
no, imo, it doesnt do anything at all, in the __call__ func we try to get it anyways.

Reply:
Please leave it if you are unsure.

Reply:
```lnbits/decorators.py:57: error: Incompatible types in assignment (expression has type "Optional[Wallet]", variable has type "None")
```